src/main/java/ru/skypro/homework/controller/AdsController.java

```diff
@@ -0,0 +1,54 @@
+package ru.skypro.homework.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import ru.skypro.homework.dto.AdDTO;
+import ru.skypro.homework.dto.AdsDTO;
+import ru.skypro.homework.dto.CreateOrUpdateAdDTO;
+import ru.skypro.homework.dto.ExtendedAdDTO;
+
+@CrossOrigin(value = "http://localhost:3000")
+@RestController
+@RequiredArgsConstructor
+public class AdsController {
+
+    @GetMapping(" /ads")
+    public ResponseEntity<AdsDTO> getAllAds() {
+        return ResponseEntity.ok(new AdsDTO());
+    }
+
+    @PostMapping("/ads")
```

Comment:
```suggestion
    @PostMapping(value = "/ads", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
```
вот так должно быть, иначе не будет работать